feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/data/source/providers/uniques/UniquesNftProvider.kt

```diff
@@ -180,4 +215,17 @@ class UniquesNftProvider(
     private fun identifier(chainId: ChainId, collectionId: BigInteger, instanceId: BigInteger): String {
         return "$chainId-$collectionId-$instanceId"
     }
+    
+    private fun bindNftOwnerAddress(scale: String?, runtime: RuntimeSnapshot): String {
+        return scale?.let { bindOrmlAccountData(it, runtime) } ?: ""
+    }
+
+    @UseCaseBinding
+    fun bindOrmlAccountData(scale: String, runtime: RuntimeSnapshot): String {
+        val type = runtime.metadata.uniques().storage("Asset").returnType()
+
+        val dynamicInstance = type.fromHexOrNull(runtime, scale).cast<Struct.Instance>()
+
+        return bindAccountId(dynamicInstance["owner"]).toHexString()
```

Comment:
I noticed that we perform toHextString() conversion here as well as in `subscribeOwnernftAddresses`. I suggest to compare `accountId` directly to avoid unnesessary conversion of accountId to hex

Reply:
Fixed